<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageMouse</class>
 <widget class="QWidget" name="pageMouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>653</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,1">
   <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cursor Theme</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="cursorTheme"/>
   </item>
   <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Cursor Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QSpinBox" name="cursorSize"/>
   </item>
   <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Natural Scroll</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="naturalScroll">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_leftHanded">
     <property name="text">
      <string>Left Handed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="leftHanded">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_pointerSpeed">
     <property name="text">
      <string>Pointer Speed</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QSlider" name="pointerSpeed">
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_accelProfile">
     <property name="text">
      <string>Acceleration Profile</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="accelProfile">
     <item>
      <property name="text">
       <string>flat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>adaptive</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_sendEventsMode">
     <property name="text">
      <string>Touchpad</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="sendEventsMode">
     <item>
      <property name="text">
       <string>enabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>disabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>disabled-on-external-mouse</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_tap">
     <property name="text">
      <string>Tap to Click</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="tap">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_tapButtonMap">
     <property name="text">
      <string>Tap Button Map</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="tapButtonMap">
     <item>
      <property name="text">
       <string>lrm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>lmr</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_tapAndDrag">
     <property name="text">
      <string>Tap and Drag</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="tapAndDrag">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="3" alignment="Qt::AlignmentFlag::AlignRight">
    <widget class="QLabel" name="label_dragLock">
     <property name="text">
      <string>Drag Lock</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="dragLock">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="10" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_threeFingerDrag">
     <property name="text">
      <string>Three Finger Drag</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="threeFingerDrag">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_middleEmulation">
     <property name="text">
      <string>Middle Button Emulation</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="middleEmulation">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_disableWhileTyping">
     <property name="text">
      <string>Disable While Typing</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QCheckBox" name="disableWhileTyping">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>33</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="13" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_clickMethod">
     <property name="text">
      <string>Click Method</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="clickMethod">
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>buttonareas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>clickfinger</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_scrollMethod">
     <property name="text">
      <string>Scroll Method</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QComboBox" name="scrollMethod">
     <item>
      <property name="text">
       <string>twofinger</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>edge</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>onbuttondown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="0" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QLabel" name="label_scrollFactor">
     <property name="text">
      <string>Scroll Factor</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="4" alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QDoubleSpinBox" name="scrollFactor">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
